docs/network-flow-visibility.md

```diff
@@ -171,6 +189,118 @@ through [Antrea Agent apiserver endpoint](prometheus-integration.md):
+  # Flow export interval should be greater than or equal to 1s (one second).
+  # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  #flowExportInterval: 60s
+  
+  # Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
+  #aggregatorTransportProtocol: "tcp"
+```
+
+Please note that the default values for `flowExportInterval` and `aggregatorTransportProtocol`
+parameters are set to `60s` and `tcp`, respectively. Please modify them as per your
+requirements.
+
+### IPFIX Information Elements (IEs) in an Aggregated Flow Record
+
+In addition to IPFIX information elements provided in the [above section](#ipfix-information-elements-ies-in-a-flow-record),
+the Flow Aggregator adds following fields to the flow records.
+
+#### IEs from IANA-assigned IE registry
+
+| IPFIX Information Element  | Enterprise ID | Field ID | Type        |
+|----------------------------|---------------|----------|-------------|
+| originalExporterIPv4Address|      0        |   403    | ipv4Address |
+| originalObservationDomainId|      0        |   151    | unsigned32  |
+
+#### IEs from Antrea IE Registry
+
+| IPFIX Information Element                 | Enterprise ID | Field ID | Type        |
+|-------------------------------------------|---------------|----------|-------------|
+| packetTotalCountFromSourceNode            | 56506         | 120      | unsigned64  |
+| octetTotalCountFromSourceNode             | 56506         | 121      | unsigned64  |
+| packetDeltaCountFromSourceNode            | 56506         | 122      | unsigned64  |
+| octetDeltaCountFromSourceNode             | 56506         | 123      | unsigned64  |
+| reversePacketTotalCountFromSourceNode     | 56506         | 124      | unsigned64  |
+| reverseOctetTotalCountFromSourceNode      | 56506         | 125      | unsigned64  |
+| reversePacketDeltaCountFromSourceNode     | 56506         | 126      | unsigned64  |
+| reverseOctetDeltaCountFromSourceNode      | 56506         | 127      | unsigned64  |
+| packetTotalCountFromDestinationNode       | 56506         | 128      | unsigned64  |
+| octetTotalCountFromDestinationNode        | 56506         | 129      | unsigned64  |
+| packetDeltaCountFromDestinationNode       | 56506         | 130      | unsigned64  |
+| octetDeltaCountFromDestinationNode        | 56506         | 131      | unsigned64  |
+| reversePacketTotalCountFromDestinationNode| 56506         | 132      | unsigned64  |
+| reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
+| reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
+| reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
+
+### Supported capabilities
+
+#### Storage of Flow Records
+
+Flow Aggregator stores the received flow records from Antrea Agents in a hash map,
+where the flow key is five-tuple of a network connection. Five-tuple consists of Source IP,
```

Comment:
I have never seen it spelled "five-tuple" :)

Reply:
Thought that would be more formal. Just did a search with "five-tuple" on google scholar, and there are 10K results :)
Changed to "5-tuple".. the easier convention.